packages/o-spreadsheet-engine/src/helpers/pivot_table_presets.ts

```diff
@@ -0,0 +1,469 @@
+import { TableStyle } from "../types/table";
+import { darkenColor } from "./color";
+import { TABLE_COLOR_SETS, TABLE_PRESETS, TableStyleTemplate } from "./table_presets";
+
+const pivotLightWithLightBorders: TableStyleTemplate = (colorSet) => ({
+  category: "light",
+  templateName: "pivotLightWithLightBorders",
+  primaryColor: colorSet.highlight,
+  wholeTable: {
+    style: { skipCellGridLines: true },
+    border: { horizontal: { color: colorSet.light, style: "thin" } },
+  },
+  headerRow: {
+    style: { bold: true },
+    border: {
+      top: { color: colorSet.highlight, style: "thin" },
+      bottom: { color: colorSet.highlight, style: "thin" },
+    },
+  },
+  mainSubHeaderRow: { style: { bold: true } },
+  firstSubSubHeaderRow: { style: { bold: true, textColor: colorSet.highlight } },
+  totalRow: {
+    border: {
+      top: { color: colorSet.highlight, style: "thin" },
+      bottom: { color: colorSet.highlight, style: "thin" },
+    },
+    style: { bold: true, fillColor: "#FFFFFF" },
+  },
+  firstRowStripe: {
+    style: { fillColor: colorSet.light },
+    border: {
+      bottom: { color: colorSet.medium, style: "thin" },
+      top: { color: colorSet.medium, style: "thin" },
+      vertical: { color: colorSet.medium, style: "thin" },
+    },
+  },
+  secondColumnStripe: {
+    style: { fillColor: colorSet.light },
+    border: {
+      left: { color: colorSet.medium, style: "thin" },
+      right: { color: colorSet.medium, style: "thin" },
+      horizontal: { color: colorSet.medium, style: "thin" },
+    },
+  },
+});
+
+const pivotLightWithMediumBorders: TableStyleTemplate = (colorSet) => ({
+  category: "light",
+  templateName: "pivotLightWithMediumBorders",
+  primaryColor: colorSet.highlight,
+  wholeTable: {
+    style: { textColor: colorSet.coloredText, skipCellGridLines: true },
+    border: { horizontal: { color: colorSet.light, style: "thin" } },
+  },
+  firstColumn: { border: { right: { color: colorSet.highlight, style: "thin" } } },
+  headerRow: {
+    style: { bold: true, textColor: "#000000" },
+    border: {
+      top: { color: colorSet.highlight, style: "medium" },
+      bottom: { color: colorSet.highlight, style: "medium" },
+      left: { color: colorSet.highlight, style: "medium" },
+      right: { color: colorSet.highlight, style: "medium" },
+    },
+  },
+  measureHeaderRow: { border: { top: { color: colorSet.light, style: "thin" } } },
```

Comment:
all `measureHeaderRow` seem to be unused

EDIT: or `measureHeaderRow` is supposed to be `measureHeader` ? (or the other way around)

Reply:
Yeah, renaming gone wrong ...